app/src/main/java/org/openobservatory/ooniprobe/common/PreferenceManager.java

```diff
@@ -191,6 +191,17 @@ public String getEnabledCategory() {
 		}
 	}
 
+	public ArrayList<String> getEnabledCategoryArr() {
+		ArrayList<String> list = new ArrayList<>(31);
```

Comment:
Why `31`? Can't we just leave it empty?

Reply:
that make sense, I just copied the old code that init the size at 31 made from Ale

Reply:
Oh I see:
`
When you create an object of ArrayList in Java without specifying a capacity, it is created with a default capacity which is 10. Since ArrayList is a growable array, it automatically resizes when the size (number of elements in array list) grows beyond a threshold.
`